io/plugins/eu.esdihumboldt.hale.io.xls.test/plugin.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="eu.esdihumboldt.hale.io.csv.propertyType">
```

Comment:
Probably this file was added accidentally? If so, then the build.properties above also does not need to be changed.